src/sql/connection/__init__.py

```diff
@@ -11,6 +12,7 @@
 
 __all__ = [
     "ConnectionManager",
+    "AbstractConnection",
```

Comment:
do we need this import? I don't see it used anywhere

Reply:
remove in [7ee92a9](https://github.com/ploomber/jupysql/pull/697/commits/7ee92a95c579b8fbecc5b741a15cd5aca2789ff9)